ios/FluentUI.Demo/FluentUI.Demo/DemoColorThemes.swift

```diff
@@ -208,4 +223,19 @@ class DemoColorGreenTheme: NSObject, ColorProviding {
         return UIColor(dynamicColor: DynamicColor(light: ColorValue(0x0F703B),
                                                   dark: ColorValue(0x60BD82)))
     }
+
+    var brandGradient1: UIColor {
+        return UIColor(dynamicColor: DynamicColor(light: ColorValue(0x107C41),
+                                                  dark: ColorValue(0x10893C)))
+    }
+    
+    var brandGradient2: UIColor {
+        return UIColor(dynamicColor: DynamicColor(light: ColorValue(0xDCF51D),
```

Comment:
Nit: if the value is the same for light and dark then there's no need to use a `DynamicColor`. Same with `brandGradient3`.